plugins/tagoio-integration/src/back/Server.ts

```diff
@@ -83,6 +85,8 @@ async function routeStartTCore(req: Request, res: Response) {
   const tcoreStartTime = new Date(Date.now() - process.uptime() * 1000);
   const osInfo = await helpers.getOSInfo();
   const profileToken = "p-".concat(req.headers.token as string);
+  const region = req.headers.region as string;
```

Comment:
Just for safety, it would be a good idea to check if `req.headers.region` is set, and if the value exists in the SDK:



On top of the file:
```ts
import { regionsDefinition } from "@tago-io/sdk/lib/regions.ts";
```

Below this line:
```ts
if (!regionsDefinition[region]) {
  // reject the request or use default region
}
```

See:
![image](https://github.com/user-attachments/assets/26b1cdce-cf88-4ff7-bdc0-b72856025f29)